workflow-templates/linting.yml

```diff
@@ -1,7 +1,11 @@
 name: Linting
-on: [push]
+on: [push, pull_request]
 jobs:
-  lint:
+  lint: 
+    # Run per push for internal contributers. This isn't possible for forked pull requests,
+    # so we'll need to run on PR events for external contributers.
+    # String comparison below is case insensitive.
+    if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.pull_request.head.repo.owner.login != 'splunk-soar-apps')
```

Comment:
could we just check that `github.event.pull_request.head.repo.fork` is true in the latter case?

Reply:
good idea, I confirmed that it works!

Reply:
oh do you still need to check that the event is a pull_request though? not sure what `github.event.pull_request` resolves to if it's not

Reply:
implicitly, the event will be a pull request the workflow is triggered either on push or pull request, and we check if the event is a push before evaluating this expression - although we may add more events